Podfile.lock

```diff
@@ -52,4 +52,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 445046ac151568c694ff286684322273f0b597d6
 
-COCOAPODS: 1.6.0
+COCOAPODS: 1.6.1
```

Comment:
We should not update the cocoapods version with this commit imho

Reply:
I'll undo this